src/System.Management.Automation/FormatAndOutput/common/DisplayDatabase/displayDescriptionData.cs

```diff
@@ -819,7 +819,7 @@ internal DisplayEntry(ExpressionToken expression)
 
             if (string.IsNullOrEmpty(Value))
                 if (Value == null || ValueType == DisplayEntryValueType.Property)
-                    throw PSTraceSource.NewArgumentNullException(nameof(expression));
```

Comment:
I wonder why do we change this? Please revert.

Reply:
We should not use `ArgumentNullException` here as the argument is not null but a property.

Reply:
Intention is _to trace_. We shouldn't remove this.

Reply:
[Documentation](https://docs.microsoft.com/dotnet/api/system.argumentnullexception) for `ArgumentNullException` states:
> An ArgumentNullException exception is thrown when a method is invoked and at least one of the passed arguments is null but should never be null.

The passed argument `expression` may not be null when we throw here so `ArgumentNullException` is not appropriate.